Resources/public/css/theme/views/testoneintro.css

```diff
@@ -0,0 +1,7 @@
+.ez-view-testoneintroview .intro__header {
+    font-size: 1.4rem;
+}
+
+.ez-view-testoneintroview.ez-view-testoneintroview--loading .ez-inline-loader {
+    display: block;
```

Comment:
The same as: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41961151